package.json

```diff
@@ -24,21 +24,24 @@
   },
   "homepage": "https://github.com/ControlX/reactjs_2_webpack#readme",
   "dependencies": {
-    "html-webpack-plugin": "^4.3.0",
-    "jest": "^26.3.0",
+    "prop-types": "^15.7.2",
     "react": "^16.13.1",
     "react-dom": "^16.13.1",
     "react-redux": "^7.2.1",
     "react-router": "^5.2.0",
-    "redux": "^4.0.5",
-    "webpack-dev-server": "^3.11.0"
+    "redux": "^4.0.5"
   },
   "devDependencies": {
     "@babel/core": "^7.11.1",
```

Comment:
You do not have **style-loader** package in dev dependencies. That's why webpack has issues with css files